src/SharedPipelineComponents/Datalock/SFA.DAS.CollectionEarnings.DataLock/DataLockProcessor.cs

```diff
@@ -37,6 +37,7 @@ protected DataLockProcessor()
         public virtual void Process()
```

Comment:
two using statements at the top of the file are unused and should be removed:

using System.Collections.Generic;
using System.Linq;